Update dskit to support gzip-compressed runtime configs (#9074)
grafana/mimir: CHANGELOG.md

* [ENHANCEMENT] Make `-query-frontend.additional-query-queue-dimensions-enabled` and `-query-scheduler.additional-query-queue-dimensions-enabled` non-operational flags in preparation for removal. #8984
* [ENHANCEMENT] Add a new ingester endpoint to prepare instances to downscale. #8956
* [ENHANCEMENT] Query-scheduler: Add `query-scheduler.prioritize-query-components` which, when enabled, will primarily prioritize dequeuing fairly across queue components, and secondarily prioritize dequeuing fairly across tenants. When disabled, tenant fairness is primarily prioritized. `query-scheduler.use-multi-algorithm-query-queue` must be enabled in order to use this flag. #9016 #9071

> * [ENHANCEMENT] Runtime configuration can now read gzip-compressed files with `.gz` extension. #9074

Reviewer: ```suggestion
* [ENHANCEMENT] Update runtime configuration to read gzip-compressed files with `.gz` extension. #9074
```
